build.gradle

```diff
@@ -28,6 +28,11 @@ springBoot {
     buildInfo()
 }
 
+test {
+    testLogging.showStandardStreams = true
```

Comment:
We are getting a lot of builds hanging so stuck this in to find out at what point it hangs.

Reply:
I think there is one test - around documents - which takes a really long time, and even fails 50% of the time for me. Circle usually succeeds. Might even be worth taking this test out for now?

Reply:
I take a look. Thanks